.cargo/config

```diff
@@ -8,16 +8,16 @@ runner = './test_runner.sh'
 git-fetch-with-cli = true
 
 [target.x86_64-unknown-linux-gnu]
```

Comment:
Are these even needed any more? I think now it's all default targets

Reply:
At least some of them are, because I saw the build failing without them. TBH I do not want to investigate it now, because I want to merge this one ASAP. I've created LLT-5117 for it